perf/bps_tree.cc

```diff
@@ -494,22 +581,26 @@ test_insert_rand(benchmark::State &state, size_t count)
 	test_insert<tree>(state, count, RandomKey(count));
 }
 
-generate_benchmarks_size_iterations(insert_rand, 10000000);
+generate_benchmarks_size_iterations(insert_rand, 1000000);
```

Comment:
Please mention in the commit message that you reduce the data set size or, even better, do it in a separate patch with a proper explanation.

Reply:
Created a separated commit for that.